src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -572,6 +578,17 @@ def _parse_requests(self, requests: dict) -> LLMMetrics:
             num_input_tokens,
         )
 
+    def _scale_timestamps(
```

Comment:
This function is doing two things. You should be able to call this twice, once with req_timestamps and once with res_timestamps. If you do that, you can rename it to something more intuitive, I think, like "_convert_ns_to_ms". Another level up refactor-wise, you could just have a constant e.g. NANOSECONDS_IN_MILLISECONDS that you multiply or divide by when you need to do these conversions so that you don't need another function and have all the logic in one line.